docs/metrics.asciidoc

```diff
@@ -174,4 +174,70 @@ format: ms
 Platform: all.
 
 The approximate accumulated collection elapsed time in milliseconds.
+
+[float]
+[[metrics-application]]
+=== Built-in application metrics
+
+To power the {apm-app-ref}/transactions.html[Time spent by span type] graph,
+the agent collects summarized metrics about the timings of spans and transactions,
+broken down by span type.
+
+*`transaction.duration`*::
++
+--
+type: simple timer
+
+This timer tracks the duration of transactions and allows for the creation of graphs displaying a weighted average.
+
+Fields:
+
+* `sum.us`: The sum of all transaction durations in ms since the last report (the delta)
```

Comment:
Hahh - I looked into this and indeed there was an issue here. 

>  is the `u` in `.us` intended to be `μ` (mu)?

Yes! 

> but the description says ms, milliseconds. Is that correct?

No, it wasn't. This was a copy from the java agent docs, and in fact we sent the data in milliseconds in the PR originally. I looked into more docs and this clearly should be microseconds (e.g. the Go agent docs has this correctly). I fixed the code, so now we send it in microseconds and I also fixed this part of the doc as well - all microseconds now.

Side-note: since the breakdown metrics are used to calculate the time spent in different span types by percentage, I think the unit will not break the feature as long as all the data is in the same unit (which was the case), so likely that's why it was not an issue before - it'd be nice to see if all agents send it in the right unit :)